### PR TITLE
Add separate Dev Services section

### PR DESCRIPTION
**Pull request by kdubois**
Adding a new section for Dev Services.  I moved it to below the kubernetes deploy section so as to not make the initial flow more complicated.  I also moved the Spring section lower down since it's not part of the basics flow either.  Let me know what you think.

### CODE REVIEW

File: documentation/modules/ROOT/pages/dev-services.adoc [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+We'll also need to update the configuration and change h2 to postgresql:
+
+[#quarkuspdb-update-props]
+[.console-input]
+[source,config,subs="+macros,+attributes"]
+----
+# Configuration file
+# key = value
+greeting=Hello y'all!
+quarkus.datasource.db-kind=postgresql
```
Comment (lordofthejars) [inline documentation/modules/ROOT/pages/dev-services.adoc:54]: If you only have one jdbc driver in the classpath, it's not necessary to put the property.Maybe I would add this as a TIP

File: documentation/modules/ROOT/pages/dev-services.adoc [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+    - name: pgsql
+      protocol: TCP
+      port: 5432
+      targetPort: 5432
+  type: ClusterIP
+  selector:
+    app: postgresql
+EOF
+----
+
```
Comment (lordofthejars) [inline documentation/modules/ROOT/pages/dev-services.adoc:186]: I will add the kubectl apply command explicitly.

**Review by lordofthejars — CHANGES_REQUESTED** [review]

**Review by hpgrahsl — APPROVED** [review]
THX for adding dev services and changing the nav sections! I like it.

File: documentation/modules/ROOT/pages/dev-services.adoc (hunk 1 above)
Comment (kdubois) [reply documentation/modules/ROOT/pages/dev-services.adoc:56]: good point, I was debating about leaving it in or not.  the tip makes it clear

File: documentation/modules/ROOT/pages/dev-services.adoc (hunk 2 above)
Comment (kdubois) [reply documentation/modules/ROOT/pages/dev-services.adoc:188]: @lordofthejars  You mean to create the yaml file first and then apply it ?

Comment (lordofthejars) [reply documentation/modules/ROOT/pages/dev-services.adoc:188]: Yes

Comment (kdubois) [reply documentation/modules/ROOT/pages/dev-services.adoc:188]: Ok. I made the changes, let me know

**Review by lordofthejars — APPROVED** [review]

**Pull request merged**